mantis-control-plane/mantis-control-plane-server/src/main/java/io/mantisrx/master/resourcecluster/ResourceClusterScalerActor.java

```diff
@@ -277,6 +278,12 @@ private void onTriggerClusterRuleRefreshRequest(TriggerClusterRuleRefreshRequest
         this.fetchRuleSet();
     }
 
+    private void onQueueClusterRuleRefreshRequest(QueueClusterRuleRefreshRequest req) {
+        log.info("{}: Queue a request to refresh cluster rules", this.clusterId);
+        self().tell(new TriggerClusterRuleRefreshRequest(this.clusterId), self());
+        getSender().tell(Ack.getInstance(), self());
+    }
+
```

Comment:
What's the need for a separate message? How is this different from sending an ack back as the first thing in TriggerClusterRuleRefreshRequest message handler?

Reply:
will switch to debug.